heartbeat/heartbeat.yml

```diff
@@ -55,6 +58,18 @@ setup.template.settings:
 #  env: staging
 
 
+#============================== Dashboards =====================================
```

Comment:
Are there dashboards for Heartbeat?

Reply:
Not anymore! It's all in the uptime app.

Reply:
I see what you're getting at, I should remove this section. That said, I shouldn't have included this yml update in this PR, it's generated from a script. I'll remove the file from here and open another PR for that.